db_schema/migrations/12_hoster_support.sql

```diff
@@ -0,0 +1,55 @@
+INSERT INTO migrations (migration_id, migration_name) VALUES (12, 'genericHoster');
+
+CREATE TABLE IF NOT EXISTS hosterPodcastMetadata (
+  account_id INTEGER NOT NULL,
+  hoster_id MEDIUMINT UNSIGNED NOT NULL,
+  date DATE NOT NULL,
```

Comment:
if it is just a last changed, we could do a changed timestamp and ON UPDATE UPDATE TIMESTAMP?